src/wallet/sqlite.cpp

```diff
@@ -395,7 +400,7 @@ SQLiteBatch::SQLiteBatch(SQLiteDatabase& database)
 void SQLiteBatch::Close()
 {
     // If m_db is in a transaction (i.e. not in autocommit mode), then abort the transaction in progress
```

Comment:
In commit "sqlite: introduce HasActiveTxn method" (5670a21f5ef28748b6b7242c03b82f39dcbc983f)

Would be good to drop the parenthetical about autocommit mode here, since that's now an implementation detail of HasActiveTxn()

Reply:
done